src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FFIDMetadataRouteSpec.scala

```diff
@@ -101,12 +101,14 @@ class FFIDMetadataRouteSpec extends AnyFlatSpec with Matchers with TestRequest w
   private def resetDatabase(): Unit = {
     DbConnection.db.source.createConnection().prepareStatement("delete from FFIDMetadataMatches").executeUpdate()
     DbConnection.db.source.createConnection().prepareStatement("delete from FFIDMetadata").executeUpdate()
+    DbConnection.db.source.createConnection().prepareStatement("delete from FileMetadata").executeUpdate()
+    DbConnection.db.source.createConnection().prepareStatement("delete from FileProperty").executeUpdate()
     DbConnection.db.source.createConnection().prepareStatement("delete from File").executeUpdate()
     DbConnection.db.source.createConnection().prepareStatement("delete from Consignment").executeUpdate()
   }
 
   private def checkNoFFIDMetadataAdded(): Unit = {
-    val sql = "select * from FileMetadata;"
+    val sql = "select * from FFIDMetadata;"
```

Comment:
Upper case commands "SELECT" and "FROM" for consistency?

Reply:
And this one